docs/modules/tiered-storage/pages/overview.adoc

```diff
@@ -7,22 +7,26 @@
 
 {description}
 
-NOTE: If your Enterprise license was generated before Hazelcast Platform version 5.2, you'll need a new Enterprise license that enables Tiered Storage. See xref:deploy:enterprise-licenses.adoc[].
+NOTE: If your Enterprise license was generated before Hazelcast Platform version 5.2, you'll need a new Enterprise license that enables Tiered Storage.
+See xref:deploy:enterprise-licenses.adoc[].
 
 == What Can You Do with Tiered Storage?
 
 Have fewer Hazelcast members for large maps. For example, for a map with 1 TB of data, you'll need
```

Comment:
Suggestion: Remove 'what can you do with TS' and amalgamate these two sections.

Tiered Storage allows you to work with datasets that are larger than the available memory using the same Hazelcast map API. It allows you to reduce infrastructure costs by utilizing cheaper disk storage and reducing the number of Hazelcast cluster members deployed.

Reply:
Made an attempt in 06a2a152059bfb69e1201dfdbf7842c9b65f589f, please take a look if that's what you meant.